indico/web/templates/forms/principal_list_widget.html

```diff
@@ -2,8 +2,13 @@
 
 
 {% block html %}
+    {% if acl %}
+        <a href="#" class="enable-acl-link hidden" title="{% trans %}Protected mode will be enabled{% endtrans %}">
+            {%- trans %}Enable access control list{% endtrans -%}
```

Comment:
I would suggest `Restrict access to this event` with a tooltip `This will change the event protection mode to "Protected"`